blocks/api/serializer.js

```diff
@@ -25,7 +25,10 @@ import BlockContentProvider from '../block-content-provider';
  * @return {string} The block's default class.
  */
 export function getBlockDefaultClassname( blockName ) {
-	// Drop common prefixes: 'core/' or 'core-' (in 'core-embed/')
+	// Generated HTML classes for blocks follow the `wp-block-{name}` nomenclature.
+	// Blocks provided by WordPress drop the prefixes 'core/' or 'core-' (used in 'core-embed/').
+
+	// @todo This should be filterable.
```

Comment:
@gziolo this seems like a good thing to make filterable.

Reply:
At the moment you can disable the default class name using `supports` and `className`:
```js
// Remove the support for a the generated className .
className: false,
```
See more https://github.com/WordPress/gutenberg/blob/master/docs/block-api.md#supports-optional.

Yes, it seems like we might make it possible to override this function with a custom implementation. I'll open an issue.

Reply:
Done: #5299 👍